packages/react-sdk/src/state/whiteboardManagerImpl.test.ts

```diff
@@ -24,7 +25,9 @@ let widgetApi: MockedWidgetApi;
 
 afterEach(() => widgetApi.stop());
 
-beforeEach(() => (widgetApi = mockWidgetApi()));
+beforeEach(() => {
+  widgetApi = mockWidgetApi();
+});
 
 const createTestWhiteboardManager = () => {
```

Comment:
for some reason here the widgetApi promise is resolved
```
const createTestWhiteboardManager = () => {
  const store = createStore({ widgetApi });
  return createWhiteboardManager(store, Promise.resolve(widgetApi));
};
```

while createWhiteboardManager expects a promise:
```
export function createWhiteboardManager(
  store: StoreType,
  widgetApiPromise: Promise<WidgetApi>,
): WhiteboardManager {
```

Reply:
Promise.resolve returns a Promise as expected. It just means it will immediately return a type when awaited.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/resolve